additional-functions/manage-restrictions.md

```diff
@@ -0,0 +1,49 @@
+## Manage restrictions
+
+Restrictions in Mews are configured as combinations of [conditions and exceptions](https://help.mews.com/en/articles/4244406-how-restrictions-work). Below are the three [restriction types](../operations/services.md#restriction-type) that can be applied:
```

Comment:
These three paragraphs IMHO stand for missing documentation or incomplete documentation of the restriction entity and some of the restriction endpoints. I'd rather extend that because that has much bigger impact than having it here.